opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -51,6 +52,7 @@ where
     ///
     /// If this option is not used or `interval` is equal to zero, 60 seconds is
     /// used as the default.
+    #[allow(unused)]
```

Comment:
It's common for users to override the default export interval. This should be made available to users by default without having to opt-in for any feature.

Reply:
Based on offline discussions, I kept periodic_reader back as public, so existing behaviors would continue to work as-is.